README.rst

```diff
@@ -334,3 +334,74 @@ If you want to send multiple types of grade back, that can be done by specifying
     ags.put_grade(gr, line_item);
 
 If a lineitem with the same ``tag`` exists, that lineitem will be used, otherwise a new lineitem will be created.
+
+Usage with Flask
+================
+
+Open Id Connect Login Request
+-----------------------------
+
+This is draft of API endpoint. Wrap it in library of your choice.
+
+Create ``FlaskRequest`` adapter. Then create instance of ``FlaskOIDCLogin``. ``redirect`` method will return instance of ``werkzeug.wrappers.Response`` that points to LTI platform if login was successful. Handle exceptions.
+
+.. code-block:: python
+
+    from flask import request, session
+    from pylti1p3.flask_adapter import (FlaskCookieService, FlaskOIDCLogin,
+                                        FlaskRequest, FlaskSessionService)
+
+    def login(request_params_dict):
+
+        tool_conf = ... # See Configuration chapter above
+
+        request = FlaskRequest(
+            request=request_params_dict,
+            cookies=request.cookies,
+            session=session
+        )
```

Comment:
Thanks for detailed documentation! It looks awesome! 👍  I have one question regarding `FlaskRequest` params. Posted it below.